python/mlc_chat/op/position_embedding.py

```diff
@@ -141,16 +141,16 @@ def fused_rope(  # pylint: disable=too-many-locals
             with T.block("llama_fused_rope"):
                 b, s, h, d = T.axis.remap("SSSS", iters)
                 if h < num_q_heads:
-                    q[b, s, h, d] = (
-                        _rope(qkv, b, s, h, d, total_seq_len - seq_len)
-                        if d < rotary_dim
-                        else qkv[b, s, h, d]
```

Comment:
`B if A else C` does not naturally work with TIR parser.

Reply:
Seems something we should fix in the parser :))